### ISSUE TITLE
Make drag & drop accessible for modifying schema

### ISSUE THREAD

**Issue by jenweber** [cardstack/cardstack#1198]
Someone should be able to use a keyboard only in order to modify their card. There are some established patterns for this. This is a nice overview: [Can drag and drop be made accessible? Yes!](https://www.barrierbreak.com/2018/10/17/can-drag-and-drop-be-made-accessible-yes-it-can-be-in-different-ways/)

**Comment by jenweber** [comment]
Will started the work for the keyboard shortcuts, so have a look to see what code is already there before you get started. See https://github.com/cardstack/cardstack/pull/1314
